workflow/v1/README.md

```diff
@@ -0,0 +1,167 @@
+[`release`]:
+  https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release
+[`push`]:
+  https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#push
+[`workflow_dispatch`]:
+  https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch
+
+This build type MUST NOT be used for any other event type unless this
+specification is first updated. Other event types are irrelevant for software
+builds (such as `issues`) or have complex semantics that may be error prone or
+require further analysis (such as `pull_request` or `repository_dispatch`). To
+add support for another event type, please open a [GitHub Issue].
+
+Note: Consumers SHOULD reject unrecognized external parameters, so new event
+types can be added without a major version bump as long as they do not change
+the semantics of existing external parameters.
+
+[GitHub Issue]: https://github.com/actions/buildtypes/issues
+[GitHub Actions]: https://docs.github.com/en/actions
+[event types]:
+  https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows
+
+## Build Definition
+
+### External parameters
+
+[External parameters]: #external-parameters
+
+All external parameters are REQUIRED unless empty.
+
+<table>
+<tr><th>Parameter<th>Type<th>Description
+
+<tr id="workflow"><td><code>workflow</code><td>object<td>
+
+The workflow that was run.
+
+<tr id="workflow.ref"><td><code>workflow.ref</code><td>string<td>
+
+A git reference to the commit containing the workflow, as either a git ref
+(starting with `refs/`). This is the value passed in via the event.
+
+Can be computed from the [github context] using `github.ref`. Note that
+`github.ref` is not guaranteed to be available for all event types but should be
+present for all currently supported event types.
+
+<tr id="workflow.repository"><td><code>workflow.repository</code><td>string<td>
+
+HTTPS URI of the git repository, with `https://` protocol and without `.git`
+suffix.
+
+Can be computed from the [github context] using
+`github.server_url + "/" + github.repository`.
+
+<tr id="workflow.path"><td><code>workflow.path</code><td>string<td>
+
+The path to the workflow YAML file within the commit.
+
+Can be computed from the [github context] using `github.workflow_ref`, removing
+the prefix `github.repository + "/"` and the suffix `"@" + github.ref`. Take
+care to consider that the path and/or ref MAY contain `@` symbols.
+
+</table>
+
+[github context]:
+  https://docs.github.com/en/actions/learn-github-actions/contexts#github-context
+[release body parameters]:
+  https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release--parameters
+[variables]: https://docs.github.com/en/actions/learn-github-actions/variables
+[vars context]:
+  https://docs.github.com/en/actions/learn-github-actions/contexts#vars-context
+
+### Internal parameters
+
+All internal parameters are OPTIONAL.
```

Comment:
I think we need to update the section on which parameters are optional. The `github` object MUST be present, and the listed elements MUST also be present. Other elements are OPTIONAL.